packages/frontend/src/components/accounts/AccountListImport.js

```diff
@@ -0,0 +1,215 @@
+                40% {
+                    color: #ae6816;
+                    text-shadow:
+                        .3em 0 0 rgba(0,0,0,0),
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                60% {
+                    text-shadow:
+                        .3em 0 0 #ae6816,
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                80%, 100% {
+                    text-shadow:
+                        .3em 0 0 #ae6816,
+                        .6em 0 0 #ae6816;
+                }
+            }
+        }
+    }
+    &.pending .status {
+        background: #f4c898;
+        color: #ae6816;
+        text-align: left;
+        padding: 0 0 0 10px;
+        flex: 0 0 82px;
+
+        :after {
+            content: '.';
+            animation: dots 1s steps(5, end) infinite;
+        
+            @keyframes dots {
+                0%, 20% {
+                    color: rgba(0,0,0,0);
+                    text-shadow:
+                        .3em 0 0 rgba(0,0,0,0),
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                40% {
+                    color: #ae6816;
+                    text-shadow:
+                        .3em 0 0 rgba(0,0,0,0),
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                60% {
+                    text-shadow:
+                        .3em 0 0 #ae6816,
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                80%, 100% {
+                    text-shadow:
+                        .3em 0 0 #ae6816,
+                        .6em 0 0 #ae6816;
+                }
+            }
+        }
+    }
+    &.waiting {
+        .status {
+            background: #f8f8f8;
+            color: #aaaaaa;
+        }
+        > div:first-of-type {
+            opacity: 0.4;
+        }
+        h3 {
+            color: #aaaaaa !important;
+        }
+    }
+
+    .right-arrow {
+      width: 24px;
+      height: 24px;
+    }
+
+    .status {
+        flex: 0 0 72px;
+        margin-left: auto;
+        height: 24px;
+        border-radius: 12px;
+        text-align: center;
+        font-size: 12px;
+        line-height: 24px;
+    }
+}
+`;
+
+const AccountListImport = ({ accounts = [], animationScope = 0, onClickAccount }) => (
+  <AnimateList animate={animationScope}>
+      {accounts.map((account) => (
+          <div
+            key={account.accountId}
+            className={`row ${account.status}`}
+            onClick={() => onClickAccount ? onClickAccount(account) : null}
+            style={{ cursor: onClickAccount ? 'pointer' : 'default' }}
+          >
+              <UserIcon>
+                  <UserIconGrey color='#9a9a9a' />
+              </UserIcon>
+              <div className='accountId'>
+                  {account.accountId}
+              </div>
+              {onClickAccount ? <div className='status onclick'><IconArrowRight stroke="#0072CE" /></div> : null}
```

Comment:
No strong opinion here but had the idea that ternaries were recommended in this case since some falsy values will render. e.g. `0 && <div className='status onclick'><IconArrowRight stroke="#0072CE" /></div>` evaluates to 0? 
[react docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) 
[kent article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)
Happy to go along with the style we decide on as a team but good point to discuss either way. I've made a habit of former style 😆

Reply:
No strong opinion here but had the idea that ternaries were recommended in this case since some falsy values will render. e.g. `0 && <div className='status onclick'><IconArrowRight stroke="#0072CE" /></div>` evaluates to 0? 
[react docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) 
[kent article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)
Happy to go along with the style we decide on but good point to discuss either way. I've made a habit of former style 😆

Reply:
Great point! If the value can be numeric I'm definitely fine with ternary or coercion via `!!`.

My personal preference is to avoid ternaries on big components when the falsy case renders nothing, but it's still subjective for sure. I think there will always be exceptions when ternaries make more sense.